.npmignore

```diff
@@ -1,13 +1,17 @@
+.changes
```

Comment:
As suggested in #2341 you _might_ want to use the `files` property in `package.json`, this should prevent you from ever shipping unwanted files again :)

Reply:
@SimonSchick 

Thanks for the suggestion.  We're opting to stick with only using .npmignore for now.  We'll keep that option in mind though.